src/script/components/MessagesList/Message/ContentMessage/MessageQuote.tsx

```diff
@@ -168,7 +168,7 @@ const QuotedMessage: FC<QuotedMessageProps> = ({
       <div className="message-quote__sender">
         <button
           type="button"
-          className="button-reset-default"
+          className="button-reset-default text-left"
```

Comment:
`text-left` is a class in our mixins

Reply:
is the `button-rest-default` has a centered alignment?

Reply:
nope, just the plain old button from the user agent stylesheet